pom.xml

```diff
@@ -152,6 +154,19 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins</groupId>
+      <artifactId>configuration-as-code</artifactId>
+      <version>${jcasc.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.jenkins</groupId>
+      <artifactId>configuration-as-code</artifactId>
+      <version>${jcasc.version}</version>
+      <classifier>tests</classifier>
+      <scope>test</scope>
```

Comment:
IMHO we should detach test code to a separate module later. CC @casz @timja

Reply:
Proposed here: https://github.com/jenkinsci/configuration-as-code-plugin/issues/917